src/main/webapp/user.html

```diff
@@ -0,0 +1,87 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.1.0/css/bootstrap.min.css"
+    integrity="sha384-9gVQ4dYFwwWSjIDZnLEWnxCjeSWFphJiwGPXr1jddIhOegiu1FwO5qRGvFXOdJZ4" crossorigin="anonymous">
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.0/umd/popper.min.js"
+    integrity="sha384-cs/chFZiN24E4KMATLdqdvsezGxaGsi4hLGOzlXwp5UZB1LY//20VyM2taTB4QvJ"
+    crossorigin="anonymous"></script>
+  <link rel="stylesheet" type="text/css" href="style.css">
+  <script src="https://code.jquery.com/jquery-3.2.1.min.js" crossorigin="anonymous"></script>
+  <script defer src="https://use.fontawesome.com/releases/v5.0.13/js/solid.js"
+    integrity="sha384-tzzSw1/Vo+0N5UhStP3bvwWPq+uvzCMfrN1fEFe+xBmv1C/AtVX5K0uZtmcHitFZ"
+    crossorigin="anonymous"></script>
+  <script defer src="https://use.fontawesome.com/releases/v5.0.13/js/fontawesome.js"
+    integrity="sha384-6OIrr52G08NpOFSZdxxz1xdNSndlD4vdcf/q2myIUVO0VsqaGHJsB0RaBE01VTOY"
+    crossorigin="anonymous"></script>
+  <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.1.0/js/bootstrap.min.js"
+    integrity="sha384-uefMccjFJAIv6A+rW+L4AHf99KvxDjWSu1z9VI8SKNVmz4sk7buKt/6v9KI65qnm"
+    crossorigin="anonymous"></script>
+  <script src="script.js"></script>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.0/umd/popper.min.js"
+    integrity="sha384-cs/chFZiN24E4KMATLdqdvsezGxaGsi4hLGOzlXwp5UZB1LY//20VyM2taTB4QvJ"
+    crossorigin="anonymous"></script>
+</head>
+
+<body>
+
+  <div id="wrapper">
+    <!-- Used to include the same sidebar on each page.-->
+    <div id="sideBar">
+      <script>
+        $(function () {
+          $("#sideBar").load("sidebar.html");
+        });
+      </script>
+    </div>
+    <div id="page-content-wrapper">
+      <div id="user-wrapper">
+        <h1 id="user-header"></h1>
+        <div id="username-div">
+          <div id="username-placeholder">
+          </div>
+          <div id="username-setter" style="display: none;">
+            <form action="/users/username" method="POST">
+              <input type="text" placeholder="Enter a desired nickname" name="nickname" required>
+              <input type="submit" />
+            </form>
+          </div>
+        </div>
+        <div id="joined-events"><button class="btn btn-success" onclick="changePopoverColorTo('#28a745')">Click here to see all the events you joined</button>
```

Comment:
Maybe separate it to CSS?

Reply:
I can't add that to css. Each button has its own color and this color is for the tooltips. I haven't found other nicer way to make different colors to different tooltips